### DISCUSSION TITLE
Is it possible to get notified when any audio is detected?

### DISCUSSION THREAD

**Discussion by chatziko** [blakeblackshear/frigate#7798] [Ask A Question] [answered]
Hello,
audio detection sounds like an amazing adition to v0.13, thanks for implementing it!

My question: is it possible to get a notification (and/or a recording) when __any__ audio (above `min_volume`) is detected, regardless of its classification? In a sense, have the equivalent of __motion__ detection (any movement, regardless of classfication), but for audio?

**Comment by NickM-27** [comment, marked as answer]
this is already supported, if your retention mode is `motion` then any recording segment with a volume above min_volume will be retained

**Reply by chatziko** [reply]
Oh, sounds great! Is there also a corresponding "motion" sensor in HA that triggers?

Looking forward to trying it.

**Reply by NickM-27** [reply]
currently no, there are only smart audio based sensors in the HA integration. It would not be difficult to add one though

**Reply by jokerigno** [reply]
I will support creating a general audio sensor like the other detection sensors.
